.github/workflows/node-ci.yml

```diff
@@ -84,11 +84,11 @@ jobs:
       with:
         full-trigger: ${{ steps.eslint-reset.outputs.filtered }}
         full: yarn lint:ci
-        limited-trigger: ${{ steps.js-files.outputs.filtered }}
+        limited-trigger: ${{ steps.js-ts-files.outputs.filtered }}
         limited: yarn lint:ci {}
 
     - name: Typecheck
-      if: steps.js-files.outputs.filtered != '[]'
+      if: steps.js-ts-files.outputs.filtered != '[]'
       run: yarn typecheck
```

Comment:
I noticed this step wasn't running when reviewing the checks.  Upgrading the `js-files` step to `js-ts-files` should fix the issue.

Reply:
I've verified the fix by looking at the logs from the most recent workflow run.